security/cert_manager_operator/cert-manager-creating-certificate.adoc

```diff
@@ -10,19 +10,11 @@ By using the {cert-manager-operator}, you can manage certificates, handling task
 
 include::modules/cert-manager-certificate-mgmt.adoc[leveloffset=+1]
 
-include::modules/cert-manager-certificate-api-server.adoc[leveloffset=+1]
-
-include::modules/cert-manager-certificate-ingress.adoc[leveloffset=+1]
-
```

Comment:
Do we also need to remove these two files together？

https://github.com/openshift/openshift-docs/blob/enterprise-4.12/modules/cert-manager-certificate-api-server.adoc

https://github.com/openshift/openshift-docs/blob/enterprise-4.12/modules/cert-manager-certificate-ingress.adoc